automation/test-windows.sh

```diff
@@ -88,51 +85,18 @@ run_vm(){
   vm_name=$1
   template_path="dist/templates/$vm_name.yaml"
   local template_name=$( oc get -n ${namespace} -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers -n kubevirt )
-  running=false
 
-  set +e
-
-  #If first try fails, it tries 2 more time to run it, before it fails whole test
-  for i in `seq 1 3`; do
-    error=false
-
-    oc process -n $namespace -o json $template_name NAME=$vm_name DATA_SOURCE_NAME=${source_name} DATA_SOURCE_NAMESPACE=${namespace} | \
-    jq '.items[0].metadata.labels["vm.kubevirt.io/template.namespace"]="kubevirt"' | \
-    oc apply -n $namespace -f -
-    
-    # start vm
-    ./virtctl start "${vm_name}" -n "${namespace}"
-
-    oc wait --for=condition=Ready --timeout="${hour_timeout}s" "vm/${vm_name}" -n "${namespace}"
-        
-    current_time=0
-    # run command via ssh
-    while [[ $(sshpass -pAdministrator ssh -o ProxyCommand="./virtctl port-forward  \
-      --stdio=true -n ${namespace} vm/${vm_name} 33333:22" \
-      -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
-      "${username}@127.0.0.1" -p 33333 "echo Hello" | grep -c "Hello" ) != 1 ]] ; do
-      # VM can be stopped during test and recreated. That will change IP, so to be sure, get IP at every iteration
-      current_time=$((current_time + sample))
-      if [[ $current_time -gt $timeout ]]; then
-        error=true
-        break
-      fi
-      sleep $sample;
-    done
+  oc process -n $namespace -o json $template_name NAME=$vm_name DATA_SOURCE_NAME=${source_name} DATA_SOURCE_NAMESPACE=${namespace} | \
+  jq '.items[0].metadata.labels["vm.kubevirt.io/template.namespace"]="kubevirt"' | \
+  oc apply -n $namespace -f -
+  
+  # start vm
+  ./virtctl start "${vm_name}" -n "${namespace}"
 
-    delete_vm $vm_name $template_name
-    #no error were observed, the vm is running
-    if ! $error ; then
-      running=true
-      break
-    fi
-  done
-
-  set -e
+  oc wait --for=condition=Ready --timeout="${fifteen_minutes_timeout}s" "vm/${vm_name}" -n "${namespace}"
```

Comment:
IMO we can keep the hour timeout for the Ready condition, but once the VM runs it should boot within the shorter timeout.

Reply:
the vm should be ready in just few minutes, so 15 minutes is ok

Reply:
We had a timeout of 1 hour before?

Reply:
yes, because we didn't use ssd as storage. But since then, the import, start, test should be done in less than 15 minutes each